net/openvswitch/files/openvswitch.init

```diff
@@ -138,7 +155,7 @@ ovs_bridge_init() {
 	config_get name "$cfg" name $cfg
 	ovs-vsctl --may-exist add-br "$name"
 
-	config_list_foreach "$cfg" "ports" ovs_bridge_port_add
+	config_foreach ovs_bridge_port_add ovs_port
```

Comment:
The old option `ports` should continue to be functional.  It's not big burden and can be convenient compared to separate `ovs_port` section for common/basic situations.

Please consider adding a description for these options, in the package README.md

Reply:
Restored + documented.